### DISCUSSION TITLE
How to mark an OrderBookDelta as a part of OrderBookDeltas while backtesting

### DISCUSSION THREAD

**Discussion by dpmabo** [nautechsystems/nautilus_trader#1576] [General]
Hi team, we know that orderbook delta are pushed by batch at some fixed interval for most live trading(maybe for all crypto exchanges). Each delta is serialized to OrderBookDelta and save to parquet one by one. My question is how to tell the engine that an OrderBookDelta is part of an OrderBookDeltas and the deltas passed to on_order_book_deltas() is a whole OrderBookDeltas rather than a single one. This will improve backtesting performance and accuracy greatly. Thank you.

**Comment by cjdsellers** [comment]
https://discord.com/channels/924497682343550976/924506736927334400/1225025965823103037

https://github.com/nautechsystems/nautilus_trader/issues/1578
